src/trel_dnssd/trel_dnssd.hpp

```diff
@@ -176,15 +176,15 @@ class TrelDnssd
     void     RemoveAllPeers(void);
     uint16_t CountDuplicatePeers(const Peer &aPeer);
 
-    Mdns::Publisher           &mPublisher;
-    Ncp::ControllerOpenThread &mNcp;
-    TaskRunner                 mTaskRunner;
-    std::string                mTrelNetif;
-    uint32_t                   mTrelNetifIndex = 0;
-    uint64_t                   mSubscriberId   = 0;
-    RegisterInfo               mRegisterInfo;
-    PeerMap                    mPeers;
-    bool                       mMdnsPublisherReady = false;
+    Mdns::Publisher              &mPublisher;
+    Ncp::ControllerOpenThreadRcp &mController;
```

Comment:
Can we make this something simpler and shorter?

For example, is `OpenThread` really needed in the name? This is all OpenThread.

And can we use `Host` in place of `Controller`?

```suggestion
    Ncp::RcpHost &mController;
```

Reply:
Make sense! A simple name is much better.

I've made such changes:
* `ControllerOpenThreadRcp` -> `RcpHost`.
* `mController` -> `mHost`
* `aController` -> `aHost`
* `controller_openthread_rcp.hpp` -> `rcp_host.hpp`.